### DISCUSSION TITLE
Is better AMD 6700 XT 12GB or RTX 3060 12GB for generating?

### DISCUSSION THREAD

**Discussion by Chovanec** [lshqqytiger/stable-diffusion-webui-directml#155] [Q&A]
Is better AMD 6700 XT 12GB or RTX 3060 12GB for generating? What is faster?
Now I have 6700 if I change it for nvidia 3060 12gb would it be better, faster?

**Comment by Miraihi** [comment]
DirectMl is not really there at the moment to compete with Nvidia CUDA. AMD cards are just better value right now and SD DirectMl is the workaround to make them work for image generation. If you're buying the graphics card specifically for Stable Diffusion, Nvidia is a safer bet.

**Comment by Milor123** [comment]
If someone read this. Is better use a 3050 than wherever shit of AMD, i have a 6750xt and is a crapy card :(, all here MUST have a NVIDIA card

And 3060 12GB is very very faster than 6700xt, CUDA is really nice optmized, AMD dont have nothing, ROCm is other shit, less bugged than DirectML but its shit too and need linux for run it. In windows is impossible.

**Comment by Anduin1357** [comment]
I must tell you now that for AMD, only from RX 7000 series onwards that they have designed their hardware for instructions best for neural networks. You should avoid RX 6000 series for this application.

**Reply by Milor123** [reply]
For anything task we must avoid buy AMD, is stupid bad for all, bad optimized, dont have SDK for windows, ROCm have many bugs in linux, AMD is a dear crappy. 

Please guys DONT wasted you money in AMD cards, are very bad, a RTX 3050 works better than whaever shit of AMD, if you are in windows, you are DIE, DirectML works very very BAD!!

My experiencie with AMD is extremly poor, i never will buy this shits !

**Reply by Rizzlord** [reply]
If you only talk about Stable Diffusion you are right, in everything else, you are completely wrong.

**Reply by Milor123** [reply]
> If you only talk about Stable Diffusion you are right, in everything else, you are completely wrong.

Yes i understand, its true, but amd is a shit and in many aspects of work amd never will compare with work time of CUDA and other tecnolgies of NVIDIA

**Reply by Anduin1357** [reply]
> For anything task we must avoid buy AMD, is stupid bad for all, bad optimized, dont have SDK for windows, ROCm have many bugs in linux, AMD is a dear crappy.
>...

It would have been much appreciated if you didn't hijack my comment to make a thread bashing AMD.
